spring-boot-3-demo-app/build.gradle.kts

```diff
@@ -35,6 +36,7 @@ dependencies {
     testImplementation("org.springframework.kafka:spring-kafka-test")
     testImplementation("org.awaitility:awaitility")
     testImplementation("io.github.mfvanek:pg-index-health-test-starter")
+    testImplementation ("org.wiremock:wiremock-jetty12:3.10.0")
```

Comment:
Why jetty?

Reply:
Because of this https://github.com/wiremock/wiremock/issues/1928

Reply:
You have to use `testImplementation("org.springframework.cloud:spring-cloud-starter-contract-stub-runner")`
Just add BOM to the project
```
import("org.springframework.cloud:spring-cloud-dependencies:2023.0.4")
```